packages/atlas/src/views/studio/MyPaymentsView/PaymentsOverview/PaymentsOverview.hooks.ts

```diff
@@ -72,15 +81,23 @@ export const useChannelPayout = (txCallback?: () => void) => {
           return
         }
         const payloadUrl = createAssetDownloadEndpoint(nodeEndpoint, payloadDataObjectId)
-
         const { reward } = await getClaimableReward(channelId, cumulativeRewardClaimed, payloadUrl)
 
+        if (reward.gt(maxCashoutAllowed) || reward.lt(minCashoutAllowed)) {
+          setClaimError(
+            reward.gt(maxCashoutAllowed)
+              ? `Maximum limit on claimable amount of ${maxCashoutAllowed.toString()} ${TOKEN_TICKER} exceeded. Please contact creator support in Discord.`
+              : `Minimum claimable amount must be more than ${minCashoutAllowed.toString()} ${TOKEN_TICKER}. Accumulate more rewards before claiming again.`
```

Comment:
The value is in `hapi` we need to convert this to `joy`. Also, I would use `formatNumber` here: 
`formatNumber(hapiBnToTokenNumber(maxCashoutAllowed))`